src/main/windows/HistoryWindowController.js

```diff
@@ -49,6 +82,39 @@ class HistoryWindowController {
       this.window.setOpacity(0);
     });
   }
+
+  /**
+   * Sets the bounds of the window from the settings store
+   */
+  _setBounds() {
+    const currentSettings = this._Store.get("settings", DefaultSettings);
+    if (currentSettings.historyWindowBounds !== undefined) {
+      if (
+        currentSettings.historyWindowBounds.x == 0 &&
```

Comment:
Codacy found an issue: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/manual/lenny.glk/Drop/pullRequest?prid=5327605)